### ISSUE TITLE
Form elements fail colour contrast

### ISSUE THREAD

**Issue by xydlebug** [xydlebug/X-ttt#5]
Several form elements fails WCAG AA contrast requirements. Element has insufficient color contrast: foreground: #888888, background: #ffffff, font size: 13.5pt (18px), font weight: normal = 3.54 - expected >= 4.5

- On game name selection page: name input.
- On contact us form: name input, email input, subject select-option, message textarea.

See https://dequeuniversity.com/rules/axe/4.8/color-contrast?application=AxeChrome
